src/account-settings/certificate-preference/test/CertificatePreference.test.jsx

```diff
@@ -151,7 +151,7 @@ describe('NameChange', () => {
     const submitButton = screen.getByText('Choose name');
     fireEvent.click(submitButton);
     expect(mockDispatch).toHaveBeenCalledWith({
-      payload: { formId, commitValues: false },
+      payload: { formId, commitValues: false, extendedProfile: null },
```

Comment:
Please use same default value as you have used in handleSubmit because you check following in Sagas file and it will break with null
```
Object.keys(extendedProfile).length > 0 
```

Change default to empty dict
```suggestion
      payload: { formId, commitValues: false, extendedProfile: {} },
```